sanoid

```diff
@@ -29,10 +29,13 @@ if (keys %args < 2) {
 	$args{'verbose'} = 1;
 }
 
-my $pscmd = '/bin/ps';
+# for compatibility reasons, older versions used hardcoded command paths
+$ENV{'PATH'} = $ENV{'PATH'} . ":/bin:/sbin";
```

Comment:
Two questions here:

1. Is is it the task of the application to set up $PATH, or the enclosing environment?
2. Are there really accounts/configs that don't *already* have `/bin` and `/sbin` on the path?

Reply:
1) no, but it's there to be sure to not break any existing installs
2) no idea :-)

Reply:
@phreaker0 has this been tested to NOT break usage in cron with its extremely restricted default PATH on, eg, Debian and Ubuntu systems?

If we break sanoid/syncoid in cron env with their brutally restricted PATH, we're going to screw a WHOLE lot of people out of their backups until they figure out what's wrong, and it's not the simplest thing to troubleshoot. Not understanding that cron PATH is much smaller than interactive PATH has been frustrating *nix users for decades.